app/Http/Resources/ProductResource.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class ProductResource extends JsonResource
+{
+    public function toArray($request)
+    {
+        return parent::toArray($request);
```

Comment:
i think that you should return only needed fields, not all fields from query.
and if you need created_at, updated_at, you should return a human readable dateTime.

Reply:
I completely agree with @fahjan otherwise what is the benefit from it?

Reply:
Totally agreed about returning the required data manually, yet it's the front-end task to format the DateTime the way they want

Reply:
i think its a backend task, imagine if you have a 3 separated systems, for android, ios and desktop. three systems will handle date formate ?
another case, if you need to display date as a user country format D-M-Y or M-D-Y, or Hijri date.